indico/modules/events/contributions/templates/management/_contribution_report.html

```diff
@@ -123,7 +123,7 @@
                                     {%- endif %}
                                 </span>
                             </td>
-                            <td class="i-table">
+                            <td class="i-table" data-text="{{ contrib.subcontribution_count }}">
```

Comment:
sorting by subcontribution count seems a bit pointless to me - but having it also doesn't hurt anyone